package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "scripts": {
     "data-generator": "cd DataGenerator && npm start",
-    "backend": "wait-port 4003 && cd Backend && npm start",
+    "backend": "cd Backend && npm start",
+    "backend-dev": "wait-port 4003 && cd Backend && npm start dev",
     "frontend": "wait-port 4001 && cd Frontend && npm start",
-    "start": "run-p data-generator backend frontend",
+    "start": "run-p backend frontend",
+    "start-dev": "run-p data-generator backend-dev frontend",
```

Comment:
Same comment here. Also to clean this up might be worth looking at `pre`scripts and `post`scripts

Reply:
Ties in with above. I'll look into pre and post scripts though. I haven't done much with those